packages/office-ui-fabric-react/src/components/Stack/examples/Stack.Horizontal.Configure.Example.tsx

```diff
@@ -118,270 +150,204 @@ const HorizontalStackConfigureExampleContent: React.FunctionComponent<IExampleOp
-    { key: 'start', text: 'Top' },
-    { key: 'center', text: 'Center' },
-    { key: 'end', text: 'Bottom' },
-  ];
-
-  public render(): JSX.Element {
-    const { horizontalAlignment, verticalAlignment } = this.state;
-
-    return (
-      <Stack tokens={sectionStackTokens}>
-        <Stack horizontal disableShrink>
-          <Stack.Item grow>
-            <Stack>
-              <Slider
-                label="Number of items:"
-                min={1}
-                max={30}
-                step={1}
-                defaultValue={5}
-                showValue={true}
-                onChange={this._onNumItemsChange}
-              />
-              <Stack horizontal disableShrink>
-                <Checkbox
-                  label="Shadow around items"
-                  onChange={this._onBoxShadowChange}
-                  styles={shadowItemCheckboxStyles}
-                />
-                <Checkbox label="Prevent item overflow" onChange={this._onPreventOverflowChange} />
-              </Stack>
-            </Stack>
-          </Stack.Item>
-          <Stack.Item grow>
-            <Stack horizontal disableShrink tokens={configureStackTokens}>
-              <Stack>
-                <Checkbox label="Wrap items" onChange={this._onWrapChange} styles={wrapItemCheckboxStyles} />
-                <Checkbox label="Shrink items" onChange={this._onShrinkChange} />
-              </Stack>
-              <Stack.Item grow>
-                <Slider
-                  label="Container width:"
-                  min={1}
-                  max={100}
-                  step={1}
-                  defaultValue={100}
-                  showValue={true}
-                  onChange={this._onWrapperWidthChange}
-                />
-              </Stack.Item>
-            </Stack>
-          </Stack.Item>
-        </Stack>
-
-        <Stack horizontal disableShrink tokens={configureStackTokens}>
-          <Stack.Item grow>
-            <Stack>
-              <Slider
-                label="Horizontal gap between items:"
-                min={0}
-                max={50}
-                step={1}
-                defaultValue={0}
-                showValue={true}
-                onChange={this._onGapChange}
-              />
-              <Slider
-                label="Vertical gap between items:"
-                min={0}
-                max={50}
-                step={1}
-                defaultValue={0}
-                showValue={true}
-                onChange={this._onVerticalGapChange}
-              />
+  return (
+    <Stack tokens={sectionStackTokens}>
+      <Stack horizontal disableShrink>
+        <Stack.Item grow>
+          <Stack>
+            <Slider
+              label="Number of items:"
+              min={1}
+              max={30}
+              step={1}
+              defaultValue={5}
+              showValue
+              // eslint-disable-next-line react/jsx-no-bind
+              onChange={value => setNumItems(value)}
```

Comment:
Pretty sure that since `value` is the first argument you can just do `onChange={setNumItems}` (and same for all the other sliders in this and other examples)